src/drivers/uavcan/uavcan_main.cpp

```diff
@@ -552,17 +562,17 @@ UavcanNode::init(uavcan::NodeID node_id, UAVCAN_DRIVER::BusEvent &bus_events)
 	_param_restartnode_client.setCallback(RestartNodeCallback(this, &UavcanNode::cb_restart));
 
 
-	int32_t uavcan_enable = 1;
-	(void)param_get(param_find("UAVCAN_ENABLE"), &uavcan_enable);
+	int32_t uavcan_dyna_node_server = 1;
```

Comment:
Is the UAVCAN_ENABLE used at all?

Reply:
I will re-enable the use of it there. yeah, it should be used there to enable the correct use dynamic node server